can/uwrt_mars_rover_can_2021.yaml

```diff
@@ -19,20 +19,20 @@ uwrt_mars_rover_can_bus:
   - PDB
   - SCIENCE
   roboteq_canids:
```

Comment:
Could you make the canid_filter_mask and canid_filters fields in hexadecimal too?

Reply:
Wilco! I'll add that.